criu/arch/s390/crtools.c

```diff
@@ -4,6 +4,7 @@
 #include <sys/user.h>
 #include <asm/unistd.h>
 #include <sys/uio.h>
+#include <linux/const.h>
```

Comment:
Wouldn't criu/arch/s390/include/asm/restorer.h which contains the `#include <asm/ptrace.h>` be a better place for this?